app/assets/javascripts/swagger-ui/threescale.coffee.erb

```diff
@@ -100,8 +88,6 @@ class ThreeScaleHttp
       !@enableApiDocsProxy() || @desiredOrigin() == @locationOrigin()
 
   desiredOrigin: ->
-     # This is to make IE happy.
-     # https://connect.microsoft.com/IE/feedback/details/817343/ie11-scripting-value-of-htmlanchorelement-host-differs-between-script-created-link-and-link-from-document
```

Comment:
Seems that only the comment is being removed here, but not the actual code that is specific to IE.
Checking the history... https://github.com/3scale/system/commit/09ee0fa2031ae26bd2d3338204a9498b0aa136a5